### PR TITLE
netbeans: update homepage

### LINKED ISSUE #10613: [Bug]: netbeans version
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

netbeans

### Expected/Current Behaviour

Autoupdating to v17

### Possible Solution

The homepage is incorrect so checkver isn't working.

### PR DESCRIPTION
**Pull request by Zalk0**
- Update `homepage` so that checkver is working again

Closes #10613 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by StarsbySea — APPROVED** [review]
LGTM

**Comment by Zliced13** [comment]
Use this checkver instead:
```json
    "checkver": "Apache\\sNetBeans\\s([\\d]+)"
```

**Comment by Zalk0** [comment]
I forgot the comma... Sorry

File: bucket/netbeans.json [hunk 1]
```diff
@@ -57,9 +57,7 @@
             ]
         }
     },
-    "checkver": {
-        "regex": "Apache NetBeans ([\\d.]+)"
-    },
+    "checkver": "Apache\\sNetBeans\\s([\\d]+)",
     "autoupdate": {
         "url": "https://www.apache.org/dist/netbeans/netbeans/$version/netbeans-$version-bin.zip",
```
Comment (StarsbySea) [inline bucket/netbeans.json:62]:
```suggestion
        "url": "https://archive.apache.org/dist/netbeans/netbeans/$version/netbeans-$version-bin.zip",

```

Comment (StarsbySea) [reply bucket/netbeans.json:62]: Please change the autoupdate url and use the checkver.ps1 tool locally to update the version to 17

**Review by StarsbySea — CHANGES_REQUESTED** [review]

File: bucket/netbeans.json (hunk 1 above)
Comment (Zalk0) [reply bucket/netbeans.json]: Okay, I've just done it

**Comment by Zalk0** [comment]
I resolved the conflict added because of the autoupdate of the manifest that happened in the official repo, and updated my branch, so it should be good to go (should be why a merge didn't happen).

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**